docs/Alias/Registration.md

```diff
@@ -29,7 +26,7 @@ sequenceDiagram
   participant U as SSB peer
   participant R as Room server
 
-  U->>R: (muxrpc async) `room.registerAlias(alias, feedId, signature)`
+  U->>R: (muxrpc async) `room.registerAlias(alias, signature)`
   alt `alias` is an invalid alias string<br/>or already taken in the alias database<br/>or `feedId` already has an alias
```

Comment:
`feedId` removed from signature but present on this line (i remember something about maybe allowing `feedId`s to have multiple aliases? is this still true? etc)

Reply:
Oh yeah I have to remove the "or `feedId` already has an alias"